python/tchannel/tornado/connection.py

```diff
@@ -53,19 +53,15 @@
 
 class TornadoConnection(object):
     """Manages a bi-directional TChannel conversation between two machines.
-
     The following primary bi-directional operations are provided:
-
```

Comment:
Why are you removing all these lines in the docstrings? The general format used for docstrings is:

    One line summary.

    More detailed paragraph
    spanning multiple lines.

    Other paragraphs or sections in
    reStructuredText syntax, often
    separated by empty lines.

Docstrings use the reStructuredText markup syntax (see http://sphinx-doc.org/rest.html and http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html). Removing these lines breaks the formatting of the [definition lists](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#definition-lists).